jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   collectCoverage: true,
+  collectCoverageFrom: ['src/**/*.js', '!\\.test\\.js$'],
```

Comment:
Out of curiosity, doesn't this work?
```suggestion
  collectCoverageFrom: ['src/**/*.js', '!**/*test.js'],
```

Reply:
Good question! I will check

Reply:
Apparently that entry wasn't needed at all! Jest automatically skips tests from coverage reporting. I've removed this in the latest commit.